nucypher/characters/lawful.py

```diff
@@ -572,6 +573,7 @@ def retrieve(
 
         if not publisher_verifying_key:
             publisher_verifying_key = alice_verifying_key
+        publisher_verifying_key = PublicKey.from_bytes(bytes(publisher_verifying_key))
```

Comment:
I remember removing a line like that :) How do bytes turn up so deep in the call stack?

Reply:
Heh - I had the same thought.  I think it's coming from here <https://github.com/nucypher/nucypher/blob/main/examples/finnegans_wake_demo/finnegans-wake-demo-testnet-l2.py#L106>

Reply:
Let's revisit this issue in a future PR.  Other than some negligible inefficiency I don't think there's any negative impact of accepting bytes or a public key here.